apps/linode-marketplace-filecloud/roles/solr/tasks/solr-config.yml

```diff
@@ -0,0 +1,63 @@
+- name: checking if fccore is deleted
+  file: 
+    path: /opt/solrfcdata/var/solr/data/fccore
+    state: absent
+
+- name: copying fcskel into dest
+  copy:
+    src: /var/www/html/thirdparty/overrides/solarium/Solarium/fcskel
+    dest: /opt/solrfcdata/var/solr/data/
+
+- name: renaming fcskel to fccore
+  shell:
+    cmd: mv /opt/solrfcdata/var/solr/data/fcskel /opt/solrfcdata/var/solr/data/fccore
+
+- name: changing owner of fccore
+  file:
+    path: /opt/solrfcdata/var/solr/data/fccore
+    owner: solr
+    state: directory
+    recurse: yes
+
+- name: grabbing private ip
+  shell: 
+    cmd: hostname -i
+  register: private_ip
+
+- name: logging into filecloud
+  shell:
+    chdir: /tmp/
+    cmd: curl -c login.txt --location --request POST 'http://{{ private_ip.stdout }}/admin/adminlogin?adminuser=admin&adminpassword=password'
+
+- name: getting value of tonidocloud-ah
+  shell:
+    chdir: /tmp/
+    cmd: awk 'NR > 3 {print $7}' login.txt | sed -n 2p
+  register: tonidocloud_ah
+
+- name: getting value of tonidocloud-as
+  shell:
+    chdir: /tmp/
+    cmd: awk 'NR > 3 {print $7}' login.txt | sed -n 3p
```

Comment:
needs to be redone.  Use `slurp` and then regex to parse the string you are looking for.